GEOSwiftTests/MapKit+Equatable.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  MapKit+Equatable.swift
+//  GEOSwiftTests
+//
+//  Created by Andrew Hershberger on 7/22/18.
+//  Copyright © 2018 GEOSwift. All rights reserved.
+//
+
+import Foundation
+import MapKit
+
+extension CLLocationCoordinate2D: Equatable {
+    public static func == (lhs: CLLocationCoordinate2D, rhs: CLLocationCoordinate2D) -> Bool {
```

Comment:
These extensions are only included in the tests so that they won't conflict with any definitions people using this library have created for themselves. They make our tests easier to write.